### ISSUE TITLE
speed up the installation process

### ISSUE THREAD

**Issue by vncloudsco** [ail-project/ail-framework#112]
I have to wait so many days to install, Is there any way for me to shorten the time? python's library installation process takes a long time, is there any way to speed up that process? or do you guys have the full docker image for this software?

**Comment by adulau** [comment]
I did installation on a new machine and it takes less than one hour to do the installation. Not sure I get your point. What kind of system do you use?

**Comment by Terrtia** [comment]
Hi @vncloudsco !

Related to https://github.com/pypa/pip/issues/9187

Fixed on the dev branch 93cfdccc078924b303d5665f06b5ecfa89651748
We added a minimal requirement for each packages.

We are going to merge it into the master branch and do a new release in the following days

**Comment by vncloudsco** [comment]
thanks for the support i see the requirements.txt file has been updated and the installation is much faster. Previously in this file there was no version constraint, so the installation process took a long time.

For example: 

```
vouu @mtsec_gitlab Jun 16 16:18
INFO: This is taking longer than usual. You might need to provide the dependency resolver with stricter constraints to reduce runtime. If you want to abort this run, you can press Ctrl + C to do so. To improve how pip performs, tell us what happened here: https://pip.pypa.io/surveys/backtracking
  Downloading ipaddress-1.0.9.tar.gz (16 kB)
  Downloading ipaddress-1.0.8.tar.gz (16 kB)
  Downloading ipaddress-1.0.7.tar.gz (15 kB)
  Downloading ipaddress-1.0.6.tar.gz (15 kB)
  Downloading ipaddress-1.0.5.tar.gz (14 kB)
  Downloading ipaddress-1.0.4.tar.gz (14 kB)
  Downloading ipaddress-1.0.3.tar.gz (14 kB)
  Downloading ipaddress-1.0.2.tar.gz (1.1 kB)
  Downloading ipaddress-1.0.1.tar.gz (1.0 kB)
INFO: pip is looking at multiple versions of soupsieve to determine which version is compatible with other requirements. This could take a while.
Collecting soupsieve>1.2
  Downloading soupsieve-2.2-py3-none-any.whl (33 kB)
  Downloading soupsieve-2.1-py3-none-any.whl (32 kB)
  Downloading soupsieve-2.0.1-py3-none-any.whl (32 kB)
 4 replies
```
Yesterday I reported on the group chat, and it has been fixed so I will close this issue here,
